internal/cmd/gocdk/biome.go

```diff
@@ -28,37 +32,111 @@ func registerBiomeCmd(ctx context.Context, pctx *processContext, rootCmd *cobra.
 		Short: "TODO Manage biomes",
 		Long:  "TODO more about biomes",
 	}
+	var launcher string
 	biomeAddCmd := &cobra.Command{
 		Use:   "add BIOME_NAME",
 		Short: "TODO Add BIOME_NAME",
 		Long:  "TODO more about adding biomes",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(_ *cobra.Command, args []string) error {
-			return biomeAdd(ctx, pctx, args[0])
+			return biomeAdd(ctx, pctx, args[0], launcher)
 		},
 	}
+	// TODO(rvangent): There should be a way to specify answer to launcher-specific
+	// prompts via flags. Maybe a JSON snippet?
```

Comment:
This should definitely have an issue opened for it, since we've been talking about this for a while.

Reply:
Filed https://github.com/google/go-cloud/issues/2346.